Doc/library/re.rst

```diff
@@ -257,8 +257,7 @@ The special characters are:
    .. index:: single: \ (backslash); in regular expressions
 
    * Character classes such as ``\w`` or ``\S`` (defined below) are also accepted
-     inside a set, although the characters they match depends on whether
-     :const:`ASCII` or :const:`LOCALE` mode is in force.
+     inside a set, although the characters they match depends on the flags_ used.
```

Comment:
I'm not very familiar with .rst markup, but is the underscore at the end of `flags` deliberate?

Reply:
Yes, it sets a link to the Flags section. See [the specification](https://www.docutils.org/docs/ref/rst/restructuredtext.html#hyperlink-references).